requirements.txt

```diff
@@ -9,3 +9,15 @@ filelock==3.4.0
 redis==4.1.2
 pytest==7.0.1
 scipy==1.8.1
+pre-commit==2.20.0
```

Comment:
I didn't notice this in the backend PR, but why do we want to add these as dependencies? If it's just for the github actions tests, it's probably better to put them in a separate file.

Reply:
moved all linting-related dependencies to a separate ```lint_requirements.txt``` file

Reply:
If you moved them to the `lint_requirements.txt` they should be removed from this file